Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM python:3.10-slim-buster
+FROM python:3.10-slim-bullseye
 
 # copy files over
-COPY ./src /app/src
+# COPY ./src /app/src
```

Comment:
I think we need this back in

Reply:
volumes section in docker-compose.yml mounts the source now.

Reply:
yea but not in production, perhaps I can put the Testing flag in there